entries/w/wallapop.com.json

```
@@ -0,0 +1,14 @@
+{
+  "Wallapop": {
+    "domain": "wallapop.com",
+    "img": "wallapop.com.svg",
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "documentation": "https://ayuda.wallapop.com/hc/en-us/articles/7727995450129-Two-step-verification-on-Wallapop",
+    "keywords": [
+      "retail"
+    ]
```

From what I can find online, it looks like Wallapop operates in the following countries.
```suggestion
    ],
    "regions": [
      "es",
      "fr",
      "gb",
      "it",
      "pt"
    ]
```